app/src/main/res/layout/verified_wallet_chip.xml

```diff
@@ -1,48 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
+    android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    app:cardBackgroundColor="@color/balance_verify_chip_background"
-    app:cardCornerRadius="9dp"
-    app:cardElevation="0dp"
+    android:orientation="horizontal"
     >
 
-  <androidx.constraintlayout.widget.ConstraintLayout
-      android:layout_width="match_parent"
-      android:layout_height="match_parent"
-      android:orientation="horizontal"
-      android:paddingStart="4dp"
-      android:paddingEnd="4dp"
-      >
+  <ImageView
+      android:id="@+id/chip_icon"
+      android:layout_width="11dp"
+      android:layout_height="11dp"
+      android:scaleType="centerInside"
+      android:src="@drawable/ic_wallet_verified_chip"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toStartOf="@id/chip_text"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
```

Comment:
Instead of having an image view we can use the drawable property of TextViews.

Something like:

      app:drawableStartCompat="@drawable/ic_wallet_verified_chip"
      android:drawablePadding="4dp"